src/main/java/seedu/address/model/person/Period.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.model.person;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class representing a period of dates.
+ */
+public class Period {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Period should be a range of two dates with a space in between\n"
+            + "E.g. 1999-10-09 1999-11-15";
+
+    private static final String DELIMITER = " ";
+
+    private LocalDate start;
+    private LocalDate end;
```

Comment:
Maybe can change to startDate and endDate otherwise it's hard to distinguish between startTime and endTime.